packages/financial-templates-lib/src/logger/SpyTransport.ts

```diff
@@ -18,7 +18,12 @@ export class SpyTransport extends Transport {
 
   async log(info: any, callback: () => void): Promise<void> {
     // Add an `logInTest` option to help with debugging to bots in tests by printing all logs received by winston.
-    if (argv._.includes("logInTest")) console.log(JSON.stringify(info, null, 2));
+    if (argv._.includes("logInTest"))
```

Comment:
this does not work correctly if there is an error in the log. took me a while to figure out why I was not seeing the logs.